mavros_extras/CMakeLists.txt

```diff
@@ -32,8 +32,7 @@ find_package(libmavconn REQUIRED)
 
 find_package(eigen3_cmake_module REQUIRED)
 find_package(Eigen3 REQUIRED)
-# find_package(yaml_cpp REQUIRED)
-find_package(yaml_cpp_vendor REQUIRED)
+find_package(yaml-cpp REQUIRED)
```

Comment:
Probably for fresh releases vendored yaml-cpp not needed.

Reply:
At the same time it likely fixes: https://github.com/mavlink/mavros/issues/1960#issuecomment-2176126183